src/commands/synthetics/utils.ts

```diff
@@ -269,11 +269,13 @@ const getPollResultMap = async (api: APIHelper, batch: Batch) => {
 }
 
 export const getOrgSettings = async (
-  api: APIHelper,
```

Comment:
We don't export the `api` module so it's not possible to create an instance `APIHelper` externally.
So we need to create the instance _in the helper_, as we already do in `fetchTest()`.